mobile/components/MediaDialog.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ReactNode } from "react";
+import { AlertDialog, AlertDialogProps } from "tamagui";
+
+interface DialogProps extends AlertDialogProps {
+  // what you press on in order to open the dialog
+  trigger?: ReactNode;
+  // dialog header
+  header?: ReactNode;
+  // content inside dialog
+  content?: ReactNode;
+}
+
+const MediaDialog: React.FC<DialogProps> = ({ header, content, trigger, ...props }) => {
```

Comment:
I don't understand what is the difference between this and the DialogModal component that we already have?